web/helpers/validations.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Community.Validations do
+  import Ecto.Changeset
+
+  def validate_url_format(changeset, field) do
+    format = ~r/http(s)?.*/
```

Comment:
That is a good point. I would consider naming it `url_format` though since right now `format` is not that descriptive. Would that be a good compromise?